css/main.css

```diff
@@ -1,3 +1,218 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+
+body {
+  font-family: "Quicksand", sans-serif;
+  color: #2f2f2e;
+}
+
+
+ nav {
+  position: fixed;
+  width: 100%;
+  top: 0px;
+  background-color: white;
+} 
+
+.button p {
+  font-weight: lighter;
+  padding: 20px 0px;
+  background-color: #cff4ec;
+  font-family: "avenir-lt-w01_35-light1475496", sans-serif;
+  text-align: center;
+  font-weight: lighter;
+  font-size: 16px;
+}
+
+.button p:hover {
+  background-color: #4D7369;
+  color: #ffffff;
+  cursor: pointer;
+}
+
+a {
+  text-decoration: none;
+  cursor: pointer;
+  color: #2f2f2e;
+  font-size: 16px;
+}
+
+a:hover {
+  color: #4D7369;
+}
+
+.title p {
+  font-family: "Libre Baskerville", serif;
+  font-size: 21px;
+  font-style: italic;
+}
+
+div.container.main-box {
+  margin-top: 165px;
+}
+
+.main-box {
+  background-color: #cff4ec;
+  width: 100%;
+}
+
+.main-row {
+  height: 600px;
+}
+
+.main-box h1 {
+  text-align: center;
+  font-size: 58px;
+  font-family: "Libre Baskerville, serif";
+  padding: 0px 95px 50px 95px;
+  color: #2f2f2e;
+  /* font-family: Arial,Helvetica,sans-serif; */
+}
+
+.main-img {
+  padding: 0;
+}
+
+.main-box img {
+  width: 100%;
+  height: 600px;
+  object-fit: cover;
+}
+
+.bio-section {
+  background-color: #f2f2f1;
+  margin-top: 120px;
+  margin-bottom: 50px;
+}
+
+.bio-row {
+  padding-top: 100px;
+}
+
+.about-text {
+  padding-top: 15px;
+  font-family: "Libre Baskerville", serif;
+  font-style: italic;
+  font-size: 22px;
+}
+
+.bio-section p {
+  text-align: center;
+  padding-left: 80px;
+
+}
+
+button {
+  font-size:  25px;
+  border: none;
+  background-color: inherit;
+}
+ 
+
+img.img-responsive {
+  width: 100%;
+}
+
+h2 {
+  font-family: "Libre Baskerville", serif;
+  font-size: 50px;
+}
+
+.paragraph {
+  padding-top: 40px;
+}
+
+.section.therapy-options {
+  margin-top: 45px;
+  margin-bottom: 70px;
+  height: 700px;
+}
+
+.options-row {
+  height: 600px;
+  text-align: center;
+  padding-top: 50px;
+
+}
+
+.individual-therapy,
+.group-therapy,
+.couples-therapy {
+  height: 100%;
+  padding-top: 70px;
+}
+
+.individual-therapy {
+   background-color:#E2F6CD;
+}
+
+.group-therapy {
+  background-color: #f2f2f1;
+  padding-left: 30px;
+  padding-right: 30px;
+}
+
+.couples-therapy {
+  background-color:#cff4ec;
+}
+
+h3 {
+  font-family: "Libre Baskerville", serif;
+  font-size: 50px;
+  padding: 35px 0px;
+}
+
+i {
+  font-size: 25px;
+  font-family: "Libre Baskerville", serif;
+}
+
+.small-paragraph {
+  color: black;
+  padding: 0px 25px;
+}
+
+.container.article-section {
+  padding-top: 100px;
+  padding-bottom: 50px;
+}
+
+.article-section {
+  background-color: #f2f2f1;
+}
+
+h4 {
+  font-family: "Libre Baskerville", serif;
+  font-style: italic;
+  font-size: 20px;
+}
+
+.row.blog-images {
+  background-color: white;
+  width: 90%;
+  margin: 30px 0px 30px 45px;
+
+}
+
+.blog-images img {
+  width: 100%;
+  height: 200px;
```

Comment:
Don't set a width AND a height on your `img` as this will distort them. `width: 100%` should be enough, and let the height set itself naturally.
(You may need to get images that are the same aspect ratio as each other so they all line up)